geomesa-core/src/main/scala/org/locationtech/geomesa/core/util/SftBuilder.scala

```diff
@@ -149,7 +148,7 @@ object SftBuilder {
 
   // Note: not for general use - only for use with SimpleFeatureTypes parsing (doesn't escape separator characters)
   def encodeMap(opts: Map[String,String], kvSep: String, entrySep: String) =
-    opts.map { case (k, v) => (k + kvSep + v) }.mkString(entrySep)
+    opts.map { case (k, v) => k + kvSep + v }.mkString(entrySep)
```

Comment:
[picky] So, this is certainly an improvement, and it's a minor change (like the headers), but I would point out (primarily to the folks who have joined us more recently) that we typically favor one purpose per PR, yes?  Especially for large/complicated PRs.